Duplicati/CommandLine/Program.cs

```diff
@@ -48,16 +48,29 @@ public static int RealMain(string[] args)
             FROM_COMMANDLINE = true;
             try
             {
-                //If we are on Windows, append the bundled "win-tools" programs to the search path
+                //If we are on Windows, append the gpg4win path and append the bundled "win-tools" programs to the search path
                 //We add it last, to allow the user to override with other versions
                 if (Platform.IsClientWindows)
                 {
+                    var ExternalGpgPath = @"C:\Program Files (x86)\GnuPG\bin";
+                    var gpg4win = System.IO.Path.Combine(ExternalGpgPath, "gpg.exe");

```

Comment:
This path also appears in the `GetGpgExePath` method.  It would be nice if we can consolidate this by removing the duplicated code.

Reply:
Latest commit refactors the code.